hamt.go

```diff
@@ -150,21 +151,27 @@ func (n *Node) checkSize(ctx context.Context) (uint64, error) {
 }
 
 func (n *Node) Flush(ctx context.Context) error {
+	var wg sync.WaitGroup
 	for _, p := range n.Pointers {
 		if p.cache != nil {
-			if err := p.cache.Flush(ctx); err != nil {
-				return err
-			}
+			wg.Add(1)
+			go func(p *Pointer) {
+				if err := p.cache.Flush(ctx); err != nil {
+					// return err
```

Comment:
Do we need to do something with these errors now?

Reply:
oh yes we do

Reply:
and fixorated